dashboard/app/models/script.rb

```diff
@@ -518,7 +518,7 @@ def self.add_script(options, raw_script_levels)
         named_level = raw_level.delete(:named_level)
         bonus = raw_level.delete(:bonus)
         stage_flex_category = raw_level.delete(:stage_flex_category)
-        stage_lockable = raw_level.delete(:stage_lockable)
+        stage_lockable = !!raw_level.delete(:stage_lockable)
```

Comment:
Given the `!!`, can the comparison to `true` on line 595 be simplified (removed)?